style.css

```diff
@@ -107,3 +107,266 @@ a:link {
   align-items: flex-start;
   gap: 16px;
 }
+
+.works {
+  margin-top: 150px;
+}
+
+.project {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto;
+  margin-left: 30px;
+}
+
+.project_image {
+  grid-column: 1/6;
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.project_title {
+  margin: 10px 0 14px -4px;
+  grid-row-start: 2;
+  grid-column: 1/6;
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 1500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.counter {
+  width: 8px;
+  height: 8px;
+}
+
+.project_details {
+  grid-row-start: 3;
+  grid-column: 1/2;
+  display: flex;
+  align-items: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 9px;
+  line-height: 16px;
+  justify-content: space-between;
+  color: #344563;
+}
+
+.project_description {
+  margin: 8px 0 0 0;
+  grid-row-start: 4;
+  grid-column: 1/6;
+}
+
+.project > p {
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: -2px;
+}
+
+.project_lang {
+  grid-row-start: 5;
+  grid-column: 1/3;
+  display: flex;
+  align-items: flex-start;
+  margin-left: -45px;
+}
+
+.lang_type {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  padding: 4px 12px;
+  width: auto;
+  height: 24px;
+  flex: none;
+  order: 2;
+  margin: 0 4px 0 0;
+  background: #ebebff;
+  border-radius: 8px;
+  border-style: none;
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.about {
+  margin: 0 24px 0 24px;
+}
+
+.intro_about {
+  margin-top: 90px;
+  width: 250px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 5000;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.knowledge {
+  margin: 30px 0 10px 0;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 1000;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+  gap: 10px;
+}
+
+.languages {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.lang_img {
+  width: 303px;
+  height: 66.72px;
+}
+
+.myself {
+  margin-top: -45px;
+  display: flex;
+  width: 327px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
```

Comment:
- [ ] Nice one with the styling of the paragraph about the "About " Section. however, you need to adjust the font to the figma requirements 
# Figma:
![image](https://user-images.githubusercontent.com/50990925/212410750-75e1fb2b-6f26-4fba-b5d3-8cc497b7e6bf.png)

# Your Design:
![image](https://user-images.githubusercontent.com/50990925/212410819-7a6778e4-eacd-4b19-8fe9-73aa4244834e.png)

Reply:
Fonts stylings were re-downloaded from google fonts to include figma styling requirements.